src/lib.rs

```diff
@@ -113,6 +142,14 @@ where
 ///
 /// Formula (datasheet 6.2): -46.85 + 175.72 * (val / 2^16)
 /// Optimized for integer fixed point (3 digits) arithmetic.
-fn convert_temperature(temp_raw: u16) -> i16 {
-    ((((temp_raw as u32) * 21965) >> 13) - 46850) as i16
+fn convert_temperature(temp_raw: u16) -> MilliCelsius {
+    MilliCelsius(((((temp_raw as u32) * 21965) >> 13) - 46850) as i32)
+}
+
+/// Convert raw humidity measurement to thousands of a % of RH.
+///
+/// Formula (datasheet 6.1): -6 + 125 * (val / 2^16)
+/// The implementation is equivalent for fixed point (3 decimal places).
+fn raw_to_per_cent_mille_rh(humidity_raw: u16) -> RHPerCentMille {
```

Comment:
The name of this function is inconsistent with `convert_temperature`. I'd either use `convert_humidity` here, or rename both functions to `convert_raw_temperatuer` / `convert_raw_humidity`.

Reply:
done